main.tex

```diff
@@ -406,7 +406,11 @@ \subsection*{Departures from Yeadon's work}
       denominator. In this scenario, Yeadon still employs the formulae for
       stadium solids but sets the thickness to be very
       small~\cite{Yeadon1990f}.  Instead, we manipulate the equations so that
-      the approximation is not necessary.
+      the approximation is not necessary. In the case that only one of the
+      stadia for a solid has zero thickness, the divide-by-zero issue is
+      removed by computing the mass properties for the stadium solid in which
+      the two stadia are swapped. However, this manipulation does not work if
+      both stadia have zero thickness; for this case, we refer you to the code.
```

Comment:
I'm not sure if this will fly.

Is the code some crazy stuff? Why isn't an error thrown?